device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/1/port_config.ini

```diff
@@ -0,0 +1,21 @@
+# name         lanes             alias        index  role       speed       asic_port_name     coreId      corePortId      numVoq
+Ethernet144    72,73,74,75       Ethernet19/1 19     Ext        100000      Eth144-ASIC1       1           1               8
+Ethernet152    80,81,82,83       Ethernet20/1 20     Ext        100000      Eth152-ASIC1       1           3               8
+Ethernet160    88,89,90,91       Ethernet21/1 21     Ext        100000      Eth160-ASIC1       1           5               8
+Ethernet168    96,97,98,99       Ethernet22/1 22     Ext        100000      Eth168-ASIC1       1           7               8
+Ethernet176    104,105,106,107   Ethernet23/1 23     Ext        100000      Eth176-ASIC1       1           9               8
+Ethernet184    112,113,114,115   Ethernet24/1 24     Ext        100000      Eth184-ASIC1       1           11              8
+Ethernet192    120,121,122,123   Ethernet25/1 25     Ext        100000      Eth192-ASIC1       1           13              8
+Ethernet200    128,129,130,131   Ethernet26/1 26     Ext        100000      Eth200-ASIC1       1           15              8
+Ethernet208    136,137,138,139   Ethernet27/1 27     Ext        100000      Eth208-ASIC1       1           17              8
+Ethernet216    64,65,66,67       Ethernet28/1 28     Ext        100000      Eth216-ASIC1       0           19              8
+Ethernet224    56,57,58,59       Ethernet29/1 29     Ext        100000      Eth224-ASIC1       0           21              8
+Ethernet232    48,49,50,51       Ethernet30/1 30     Ext        100000      Eth232-ASIC1       0           23              8
+Ethernet240    40,41,42,43       Ethernet31/1 31     Ext        100000      Eth240-ASIC1       0           25              8
+Ethernet248    32,33,34,35       Ethernet32/1 32     Ext        100000      Eth248-ASIC1       0           27              8
+Ethernet256    24,25,26,27       Ethernet33/1 33     Ext        100000      Eth256-ASIC1       0           29              8
+Ethernet264    16,17,18,19       Ethernet34/1 34     Ext        100000      Eth264-ASIC1       0           31              8
+Ethernet272    8,9,10,11         Ethernet35/1 35     Ext        100000      Eth272-ASIC1       0           33              8
+Ethernet280    0,1,2,3           Ethernet36/1 36     Ext        100000      Eth280-ASIC1       0           35              8
+Ethernet-Rec1  221               Recirc0/0    19     Rec        400000      Rcy0-ASIC1         0           221             8
```

Comment:
The index for `Ethernet-Rec1` and `Ethernet-IB1` is overlapping with the index for `Ethernet144` and `Ethernet-152`

Reply:
Fixed